cdk/downloader_stack.py

```diff
@@ -80,7 +79,7 @@ def __init__(
             self,
             id=f"{identifier}-downloader-rds",
             engine=aws_rds.DatabaseClusterEngine.aurora_postgres(
-                version=aws_rds.AuroraPostgresEngineVersion.VER_11_15
+                version=aws_rds.AuroraPostgresEngineVersion.of("11.21", "11")
```

Comment:
@chuckwondo Will this trigger an upgrade of our Postgres engine version on upgrade?

Reply:
That's definitely a good question. Since it's a minor version bump, that shouldn't be a problem, particularly since the system is not continually running, as long as deployment occurs after the daily cron job completes, so there's no DB activity during deployment.

However, I suggest adding 1 additional step **prior to deployment: create a DB snapshot after the cron job finishes**.